### PR TITLE
Stringsifter automation

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-3263

## Description
This is an automation to activate the stringsifter module 

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by omerKarkKatz** [comment]
@yaakovi @ShirleyDenkberg  for now this PR is not for merge but POC for security

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

File: Packs/StringSifter/Scripts/StringSifter/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+This script runs the StringSifter ML tool for malware anlisys and ranking of words. You can enter an entryID or string_text as input.
```
Comment (DeanArbel) [inline Packs/StringSifter/Scripts/StringSifter/README.md:1]:
```suggestion
This script runs the StringSifter ML tool for malware analysis and ranking of words. You can enter an entryID or string_text as input.
```

File: Packs/StringSifter/Scripts/StringSifter/StringSifter.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  name: string_text
+  required: false
+  secret: false
+- default: false
+  description: The file to process.
+  isArray: false
+  name: entryID
+  required: false
+  secret: false
+comment: This script runs teh StringSifter ML tool for malware anlisys and ranking of words. You can enter an entryID or string_text as input.
```
Comment (DeanArbel) [inline Packs/StringSifter/Scripts/StringSifter/StringSifter.yml:32]:
```suggestion
comment: This script runs teh StringSifter ML tool for malware analysis and ranking of words. You can enter an entryID or string_text as input.
```

**Review by DeanArbel — APPROVED** [review]
Looks good, just 1 small note

**Pull request merged**